src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -260,37 +243,6 @@ impl DragonballInner {
             .context("put boot source")
     }
 
-    fn set_vm_rootfs(&mut self, path: &str, driver: &str) -> Result<()> {
-        info!(sl!(), "set vm rootfs {} {}", path, driver);
-        let jail_drive = self
-            .get_resource(path, DRAGONBALL_ROOT_FS)
-            .context("get resource")?;
-
-        if driver == VM_ROOTFS_DRIVER_BLK || driver == VM_ROOTFS_DRIVER_MMIO {
```

Comment:
@Tim-0731-Hzt does this check for rootfs driver happen elsewhere?

Reply:
No, I realized this is a duplicate check, as we will check it in `new_rootfs_kernel_params`, so I just remove it.